### DISCUSSION TITLE
OpenID configuration page  not showing

### DISCUSSION THREAD

**Discussion by ifoughal** [DefGuard/defguard#764] [Q&A]
Hello, I have deployed Defguard for POC purposed, and I intend to integrate the keycloack OpenID redirect.

I followed the documentation:
 https://github.com/DefGuard/docs/blob/docs/admin-and-features/external-openid-providers.md

But unfortunately, I am not sure why the OpenID tab under settings is not showing. 

Any help would be greatly appreciated as I am not sure of what I am missing.

**Comment by teon** [comment]
@ifoughal As the page states - it requires Defguard 1.0 - which is still in testing phase and should be release - around end of next week.

**Reply by ifoughal** [reply]
@teon that explains why, I have missed that particular detail, and indeed I do confirm that I am running 0.11 
Thank you for the clarification. Looking forward to 1.0!
